client/me/notification-settings/push-notification-settings/index.jsx

```diff
@@ -620,7 +620,7 @@ class PushNotificationSettings extends React.Component {
 				<div className="notification-settings-push-notification-settings__instruction-content">
 					<div>
 						<div className="notification-settings-push-notification-settings__instruction-title">
-							{ this.props.translate( 'Enable Browser Notifications' ) }
+							{ this.props.translate( 'Enable browser notifications' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.